README.md

```diff
@@ -32,12 +32,23 @@ cargo install foreman
 
 To upgrade, re-run `cargo install foreman` and clean up the `bin` directory as described in the section just above.
 
-## Usage
-Foreman downloads tools from GitHub or GitLab and references them by their `user/repo` name, like `Roblox/foreman`.
+## Setup
+Most users will want to do a bit of additional setup to begin using tools via foreman.
 
+### Path Configuration
 On first run (try `foreman list`), Foreman will create a `.foreman` directory in your user folder (usually `~/.foreman` on Unix systems, `%USERPROFILE%/.foreman` on Windows).
```

Comment:
Do we want to move that first sentence into another section? Like it could be just above in `Setup` I think